wrangler.toml

```diff
@@ -10,22 +10,19 @@ entry-point = "./workers-site"
 
 # Production
 [env.production]
-name = "docs-site"
```

Comment:
This was explicitly setting the script name to `docs-site`. env's are really scripts that inherit name from the top level env so when this line is omitted this `env.production` script's name becomes `docs-site-workers-production`